platform.txt

```diff
@@ -117,8 +118,9 @@ recipe.objcopy.eep.pattern=
 
 ## Create hex
 recipe.objcopy.hex.1.pattern="{runtime.tools.python3.path}/python3" "{runtime.tools.elf2bin}" --eboot "{runtime.tools.eboot}" --app "{build.path}/{build.project_name}.elf" --flash_mode {build.flash_mode} --flash_freq {build.flash_freq} --flash_size {build.flash_size} --path "{runtime.tools.xtensa-lx106-elf-gcc.path}/bin" --out "{build.path}/{build.project_name}.bin"
-recipe.objcopy.hex.2.pattern="{runtime.tools.python3.path}/python3" "{runtime.tools.signing}" --mode sign --privatekey "{build.source.path}/private.key" --bin "{build.path}/{build.project_name}.bin" --out "{build.path}/{build.project_name}.bin.signed" --legacy "{build.path}/{build.project_name}.bin.legacy_sig"
-recipe.objcopy.hex.3.pattern="{runtime.tools.python3.path}/python3" "{runtime.tools.sizes}" --elf "{build.path}/{build.project_name}.elf" --path "{runtime.tools.xtensa-lx106-elf-gcc.path}/bin"
+recipe.objcopy.hex.2.pattern="{runtime.tools.python3.path}/python3" "{runtime.tools.crc}" --bin "{build.path}/{build.project_name}.bin" --size 4088 --crc 4092
```

Comment:
If this is not optional, why not embed the script into the elf2bin.py?

PR is also missing PlatformIO integration because someone would need to add that script call. And while I was trying to come up with that, I am having real trouble reasoning about the dependency chain when final .bin is not final after all :/ crc32bin can have `--output` param, but that even more code

Reply:
Fair enough.  I was thinking of the UNIX philosophy (little utils which do exactly one thing).  Since systemd seems to have taken over the world, I guess I'll have to get with the times. :)  

I'll see about combining this and moving the crc32 to always be ROM safe in an upcoming push to the PR.